Formula/ctail.rb

```diff
@@ -12,15 +13,25 @@ class Ctail < Formula
     sha256 "dfab40d65950327c679bde97a335779526c58e99f5679f32f95e517a7249e332" => :mountain_lion
   end
 
-  conflicts_with "byobu", :because => "both install `ctail` binaries"
-
   depends_on "autoconf" => :build
   depends_on "automake" => :build
   depends_on "libtool" => :build
+  depends_on "apr"
+  depends_on "apr-util"
```

Comment:
`depends_on :apr` should be sufficient here for all but Sierra and will work on Sierra when https://github.com/Homebrew/brew/pull/1227 is merged shortly. Sorry about the change, it's news to me too.